Chambers_Jess/Assignments/full_friends/mysqlconnection.py

```diff
@@ -0,0 +1,40 @@
+""" import the necessary modules """
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.sql import text
+# Create a class that will give us an object that we can use to connect to a database
+class MySQLConnection(object):
+    def __init__(self, app, db):
+        config = {
+                'host': 'localhost',
+                'database': db, # we got db as an argument
+                'user': 'root',
+                'password': 'Dogen2016!',
```

Comment:
i know your password 😔 as does the whole world. i hope you don't use that same pwd for anything else. always be sure to either scrub credentials b4 committing to git, or--what i recommend is to use a dummy password like 'root' for your DB while in this class so you don't even have to worry about it.